Bottom: 9d8848694260d40f75154e420bf6ad217d107290
Top:    df93c501d6eda8afecfc7e5a466da2223351a067
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-08 02:10:05 +0000

Refresh of convet-some-safe-usage-of-myst

---

diff --git a/src/misc.c b/src/misc.c
index b87a6a6..af8cf52 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -1920,7 +1920,7 @@ AttributePutToList (AttributeListType *list, char *name, char *value, int replac
 
   /* Now add the new attribute.  */
   i = list->Number;
-  list->List[i].name = MyStrdup (name, "AttributePutToList");
+  list->List[i].name = strdup (name);
   list->List[i].value = MyStrdup (value, "AttributePutToList");
   list->Number ++;
   return 0;
